apps/loan/src/components/PageLlamaMarkets/filters/MultiSelectFilter.tsx

```diff
@@ -0,0 +1,59 @@
+import { ColumnFilter, ColumnFiltersState } from '@tanstack/react-table'
+import { Dispatch, Fragment, ReactNode, SetStateAction, useMemo } from 'react'
+import { uniq } from 'lodash'
+import Select from '@mui/material/Select'
+import Typography from '@mui/material/Typography'
+import MenuItem from '@mui/material/MenuItem'
+import { DeepKeys } from '@tanstack/table-core/build/lib/utils'
+
+// todo: doesn't this exist somewhere?
+const getByDeepKey = <T extends unknown>(obj: T, key: DeepKeys<T>) => {
```

Comment:
We use lodash right?

https://lodash.com/docs/4.17.15#get

Reply:
Aha I knew there should be something